src/adapters/search_history.js

```diff
@@ -14,6 +14,14 @@ export function getHistoryEnabled() {
   return get(SEARCH_HISTORY_KEY + '_enabled');
 }
 
+export function setHistoryPrompt(value) {
```

Comment:
This will probably be used on the next PR when the button will be clickable, but it always weirds to have code unused in a PR